admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/url/domain/URLHandlerImpl.java

```diff
@@ -45,6 +45,9 @@
 import javax.persistence.InheritanceType;
 import javax.persistence.Table;
 
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
```

Comment:
@putrenkov Still seeing import ordering issues throughout PR.

Reply:
@charchar Hi, could you check, is it better?